pkg/cloudfabric-controller/controllerframework/controller_framework.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package controllerframework
 
 import (
+	generalErrors "errors"
```

Comment:
why rename the pacakge?

Reply:
In this file there is another "errors" from community. It predefined a lot of error scenarios but I cannot find one fit my case. So I still need "errors" and rename it to avoid conflict.